### PR TITLE
Add Ocsigenserver 2.9 (OCaml 4.06 compatible)

### PR DESCRIPTION
**Pull request by vasilisp**

### CODE REVIEW

**Comment by vasilisp** [comment]
The PR failures do not look relevant.

File: packages/ocsigenserver/ocsigenserver.2.9/opam [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  "lwt" {>= "3.0.0"}
+  "lwt_ssl"
+  "lwt_react"
+  "ocamlnet" {>= "4.0.2"}
+  "pcre"
+  "cryptokit"
+  "tyxml" {>= "4.0.0"}
+  ("dbm" | "sqlite3" | "pgocaml")
+  "ipaddr" {>= "2.1"}
+  "camlp4" # to force building tyxml.parser
```
Comment (jpdeplaix) [inline packages/ocsigenserver/ocsigenserver.2.9/opam]: Crap. I missed it, sorry. @Drup is this deprecated ?

**Comment by jpdeplaix** [comment]
Fair enough. Thanks, merging now.

File: packages/ocsigenserver/ocsigenserver.2.9/opam (hunk 1 above)
Comment (Drup) [reply packages/ocsigenserver/ocsigenserver.2.9/opam]: It is, yes.

Comment (vasilisp) [reply packages/ocsigenserver/ocsigenserver.2.9/opam]: @jpdeplaix, thanks and no worries. For info, https://github.com/ocsigen/ocsigenserver/pull/124 replaces `tyxml.parser` with `xml-light` and gets rid of `camlp4`, among other old cruft. But I need to spend a few weeks on it before we finally do the migration.

**Pull request merged**